src/core/storage/compression/Compressor.h

```diff
@@ -405,36 +403,33 @@ class CompressorInternal {
 	/// \brief This function translates and writes bytes from current input file to the compressed file.
 	/// (Manages seventh of part 2)
 	///
-	/// \param original_fp - file pointer to original file
-	/// \param size - size of the original file
+	/// \param path - string that represents the path to the file/folder
 	void write_file_content(const std::string &path) {
 		const std::string buff = return_file_info(path);
 
-		for (const auto &item : buff)
-			write_bytes(m_char_huffbit[item]);
+		for (const auto &item: buff)
+			write_bytes(m_char_huffbits[(unsigned char) item]);
```

Comment:
Please use `uint8_t` rather than `unsigned char` when you mean that the value is 8-bit in size.

Reply:
Sure